src/utils/time.js

```diff
@@ -1,14 +1,18 @@
-// Build the time string manually because moment.js occasionally formats
-// the hour as 2.225071737871332e-308
-// Equivalent to `moment(date).format('h:mm A')`
-exports.formatTime = function(date) {
```

Comment:
Alright, I switched back to using `formatTime`. It accepts moment dates now, so hopefully `moment().hour` doesn't have the same problem as `.format()`.